k2/csrc/dtype.h

```diff
@@ -68,9 +68,6 @@ class DtypeTraits {
   const char *name_;  // name, e.g. "float", "int8", "int32"
 };
 
-// We initialize this in dtype.cu
-extern const DtypeTraits g_dtype_traits_array[];
```

Comment:
When building shared libs for windows, it complains that the array
`g_dtype_traits_array` is undefined.